src/client/components/pages/MultiYearPlan.tsx

```diff
@@ -79,6 +143,36 @@ const MultiYearPlan: FunctionComponent = (): ReactElement => {
       : null
   );
 
+  /**
+   * Creates the instructor filters using the multi year plan list
+   */
+  const instructorFilters = (myp: MultiYearPlanResponseDTO[]):
+  TableHeadingCell[] => (
+    myp.length > 0
+      ? myp[0].semesters
+        .map((semester: MultiYearPlanSemester, index: number):
+        TableHeadingCell => (
```

Comment:
That makes sense! I added `NUM_YEARS` and `NUM_SEMESTERS` to the `common/constants` so that these values could be used to calculate the right number of columns.